app/models/green_lanes/category_assessment_json.rb

```diff
@@ -69,6 +69,7 @@ def match?(regulation_id:, measure_type_id:, geographical_area: nil)
         measure_type_id == self.measure_type_id &&
         (geographical_area == geographical_area_id ||
           geographical_area.nil? ||
+          geographical_area.empty? ||
```

Comment:
These two should be replaced with `.blank?` which checks for nils, empty string and whitespace only strings

Reply:
done